.deploy/accessibility-app/entrypoint.sh

```diff
@@ -23,6 +23,8 @@ ln -s $CACHE_PATH /app/bootstrap/cache
 ## fix permissions after syncing to existing storage and cache https://github.com/accessibility-exchange/platform/issues/1236
 chown -R www-data:www-data $FILES_PATH $CACHE_PATH
 
+php artisan deploy:local
+
```

Comment:
Does the local deploy need to be guarded by the deploy lock? Are you able to explain a bit about what the deploy lock is preventing?

Reply:
As far as `google-fonts:fetch, icons:clear, icons:cache` I haven't dug in deep on these. They were determined by Ned that they should be done in the local one.
It all depends on the directories that they operate in. If they operate in `/app/storage` or `/app/bootstrap/cache` then I don't see why we wouldn't move them to global. I don't know if it's been tested that way.

Reply:
Do you want me to research each of these and test them?

Reply:
@marvinroman That would be fantastic. The Google Fonts command comes from https://github.com/spatie/laravel-google-fonts, and the others come from https://github.com/blade-ui-kit/blade-icons.

Reply:
@marvinroman maybe we should split off the `google-fonts:fetch, icons:clear, icons:cache` portion into a separate PR if the change is needed. In that way, we can merge the other changes here in for our upcoming release.

Reply:
@jobara I looked into fonts and icons and they are both stored in permanent storage so I recommend moving to global.

I have included those changes in this PR along with a different locking mechanism that should be more robust and I tested locally through launching 4 replicas at once and seeing global only being logged on a single one. The following command gets an exclusive lock on the `/opt/data/cache` directory which is shared across containers. It is non-blocking meaning it fails immediately if a lock exists and doesn't run the command. `-E 0` is exit code 0 which means successful since it's not an error.
```bash
flock -n -E 0 /opt/data/cache -c "php artisan deploy:global"
``` 

I have also moved the changing of file ownerships to after the caches are reset to make sure that file ownership in these directories are correct since those commands are run as root.
New global commands ordering.
```php
        $this->call('optimize:clear');
        $this->call('icons:clear');
        $this->call('event:cache');
        $this->call('icons:cache');
        $this->call('google-fonts:fetch');
        $this->call('optimize');
        $this->call('migrate', ['--step' => true, '--force' => true]);
```